packages/Tokens/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@paprika/tokens",
-  "version": "0.1.7",
+  "version": "0.1.8",
```

Comment:
This will get bumped automatically when we do the `lerna publish`, so you can leave it as `0.1.7` for now and just include a comment in the PR description that this is a "patch update" to `tokens`.